markdown/3.x/en/faq.md

```diff
@@ -122,10 +122,24 @@ If this doesn't seem to solve the issue, please contact us for a custom solution
 ---
 
 ## Errors
+#### Error:
+
+`Access to XMLHttpRequest at 'https://my.fact-finder.de/FACT-Finder/Search.ff?query=*&channel=my_channel' has been
+blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a
+preflight request.`
+
+#### Solution:
+In order to use our Web Components library, this must be activated on your FACT-Finder server.   
```

Comment:
Could we specify what exactly `this` is? The answer would become standalone and could be copy and pasted without much context.
Otherwise good 👍